doc/book/box/indexes.rst

```diff
@@ -1,18 +1,14 @@
 
---------------------------------------------------------------------------------
+================================================================================
 Indexes
---------------------------------------------------------------------------------
+================================================================================
 
-An **index** is a group of key values and pointers.
+An **index** is a special data structure that stores a group of key values and
+pointers. It is used for efficient manipulations with data.
 
 As with spaces, you should specify the index **name**, and let Tarantool
 come up with a unique **numeric identifier** ("index id").
 
-An index always has a **type**. The default index type is 'TREE'.
-TREE indexes are provided by all Tarantool engines, can index unique and
-non-unique values, support partial key searches, comparisons and ordered results.
-Additionally, memtx engine supports HASH, RTREE and BITSET indexes.
```

Comment:
It would be great to add some brief description about type - that "special data structures" could be different and should be chosen dependent on a task.